apps/dashboard/src/@3rdweb-sdk/react/hooks/useEngine.ts

```diff
@@ -118,7 +118,8 @@ type EngineFeature =
   | "CONTRACT_SUBSCRIPTIONS"
   | "IP_ALLOWLIST"
   | "HETEROGENEOUS_WALLET_TYPES"
-  | "SMART_BACKEND_WALLETS";
+  | "SMART_BACKEND_WALLETS"
+  | "WEBHOOK_CONFIG";
```

Comment:
Are the webhook CRUD APIs backward compatible, as in if config is provided, an older Engine will just silently ignore it?

If so I'm debating if it's simpler to just show the config field for all Engines (and if anything make a small footnote that config is only used for low wallet balance alerts after Engine version v2.x.x).

Reply:
Yeah old engine will not process the config as its not processed at all.

If we show it all engine, it could act as a nudge to upgrade the engine which is good I guess.